lighthouse-core/scoring.js

```diff
@@ -53,15 +53,14 @@ class ReportScoring {
           result.informative = true;
         }
 
-        return Object.assign({}, audit, {result, score: auditScore});
```

Comment:
while we're in the spirit of breaking changes, let's go whole hog and do it at the audit class level :D